### PR TITLE
More logging in debug mode

### PR DESCRIPTION
**Pull request by aidanhs**
This is probably excessively verbose (the `{!r}` prints out a lot of irrelevant info), but it's an improvement on the extreme lack at the moment.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/homu/133)

### CODE REVIEW

File: homu/main.py [hunk 1]
```diff
@@ -370,7 +371,7 @@ def verify_auth(username, repo_cfg, state, auth, realtime, my_username):
                 else:
                     reply += 'Not in reviewers'
             elif auth == AuthState.TRY:
-                    reply += 'and not in try users'
+                reply += 'and not in try users'
```
Comment (Manishearth) [inline homu/main.py]: just "not in try users"

**Comment by aidanhs** [comment]
Made change to error message.

**Comment by jdm** [comment]
```
1.25s$ flake8 homu

homu/main.py:1321:9: E722 do not use bare except'

homu/main.py:1354:9: E722 do not use bare except'

homu/server.py:698:9: E722 do not use bare except'
```

**Comment by jdm** [comment]
That seems like a new addition to flake8 that is hitting preexisting cases, unfortunately.

**Pull request merged**